src/main/java/at/hannibal2/skyhanni/config/features/chat/FilterTypesConfig.java

```diff
@@ -92,6 +92,12 @@ public class FilterTypesConfig {
     @FeatureToggle
     public boolean diana = false;
 
+    @Expose
+    @ConfigOption(name = "Factory Upgrade", desc = "Hide chocolate factory upgrade and employee promotion messages.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean factoryUpgrade = true;
```

Comment:
this should be default off i think

Reply:
I thought the messages would be annoying for most people as they would be operating inside a GUI, but I turned it off anyways